### PR TITLE
GEE-41 | add integration tests

### PR DESCRIPTION
**Pull request by jostrzol**

### CODE REVIEW

File: src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/BaseIntSpec.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    protected <T> T mapContent(byte[] content, Class<T> clazz) throws IOException {
+        return objectMapper.readValue(content, clazz);
+    }
+
+    @BeforeEach
+    protected void clearDb() {
+        testDbService.resetDatabase();
+    }
```
Comment (ianczyko) [inline src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/BaseIntSpec.java:55]: Just to make sure, it is guaranteed that these tests run sequentially?

File: src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/TestDbService.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    private List<String> tableNames() {
+        return List.of("Issues", "Issues_AUD");
+//        var metamodel = entityManagerFactory.getMetamodel();
+//        return metamodel.getEntities().stream()
+//                .map(EntityType::getJavaType)
+//                .map(clazz -> clazz.getAnnotation(Table.class))
+//                .map(Table::name)
+//                .collect(Collectors.toList());
```
Comment (ianczyko) [inline src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/TestDbService.java:40]: Is this still relevant?

**Review by ianczyko — APPROVED** [review]

File: src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/BaseIntSpec.java (hunk 1 above)
Comment (jostrzol) [reply src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/BaseIntSpec.java:55]: I think so.

File: src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/TestDbService.java (hunk 2 above)
Comment (jostrzol) [reply src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/TestDbService.java:40]: I forgor 💀

Comment (jostrzol) [reply src/integrationTest/java/pl/edu/pw/elka/paprykaisalami/geeruh/support/TestDbService.java]: Done

**Pull request merged**